File: fix.py
with open('src/pages/Students.tsx', 'r', encoding='utf-8') as f:
    code = f.read()

code = code.replace(r'\`', '`')
code = code.replace(r'\$', '$')
code = code.replace(r'\{', '{')
code = code.replace(r'\}', '}')

with open('src/pages/Students.tsx', 'w', encoding='utf-8') as f:
    f.write(code)

File: fix.js
const fs = require('fs');
let code = fs.readFileSync('src/pages/Students.tsx', 'utf8');
code = code.replace(/\\`/g, '`');
code = code.replace(/\\\$/g, '$');
code = code.replace(/\\\{/g, '{');
code = code.replace(/\\\}/g, '}');
fs.writeFileSync('src/pages/Students.tsx', code);
